### PR TITLE
Update button styling (SCRD-6919)

### PR DESCRIPTION
**Pull request by GarySmith**
Update styling of buttons to reflect the latest design from the UX team.
Changes include making the text white, changing border radius, and
changing colors of the buttons in their various states

### CODE REVIEW

File: themes/suse/_styles.scss [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 @import "./branding.scss";
 @import "./_variables.scss";
+// @import "./_button_styles.scss";
```
Comment (jeremy-moffitt) [inline themes/suse/_styles.scss]: was this a reference to something in the base theme? a file that was considered for addition but abandoned?

Comment (GarySmith) [reply themes/suse/_styles.scss]: Doh! Good catch

**Review by jeremy-moffitt — APPROVED** [review]
LGTM

**Review by dotNomad — APPROVED** [review]
Looks good :+1: . Follows the new guidelines for buttons.

**Pull request merged**